x-pack/plugins/alerting/server/task_runner/execution_handler.ts

```diff
@@ -52,6 +53,7 @@ import {
   isSummaryActionThrottled,
 } from './rule_action_helper';
 import { ConnectorAdapter } from '../connector_adapters/types';
+import { isSystemAction } from '../../common/system_actions/is_system_action';
```

Comment:
Should we use `actions.isSystemActionConnector` (exposed by the rule factory as `isSystemAction`) instead of using that helper function that checks only the `type`?